homeassistant/components/bmw_connected_drive/quality_scale.yaml

```diff
@@ -0,0 +1,80 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: |
+      Does not have custom services
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
```

Comment:
```py
    with patch(
        "bimmer_connected.api.authentication.MyBMWAuthentication.login",
        side_effect=MyBMWAuthError("Login failed"),
    ):
```
Can we patch this where we use it (so `homeassistant.components.bmw_connected_drive.config_flow.MyBMWAuthentication`)?

Reply:
Fixed by #133704.